build-tests/api-extractor-scenarios/etc/test-outputs/dynamicImportType2/rollup.d.ts

```diff
@@ -1,8 +1,10 @@
 import * as X from 'api-extractor-lib1-test';
+import * as Y from 'api-extractor-lib1-test';
```

Comment:
Update: I'm working on a better solution for this case.

Reply:
Top namespaces of a dynamic ImportType are treated now as named imports